ql/src/java/org/apache/hadoop/hive/ql/metadata/events/EventUtils.java

```diff
@@ -78,10 +79,15 @@ public long getCurrentNotificationEventId() throws IOException {
     }
 
     @Override
-    public long getDbNotificationEventsCount(long fromEventId, String dbName) throws IOException {
+    public long getDbNotificationEventsCount(long fromEventId, String dbName, Long toEventId,
```

Comment:
Why fromEventId is primitive long data type where as toEventId is Long object? Can be same.

Reply:
I see. Long object is fine then.